Makefile

```diff
@@ -13,7 +13,7 @@ OPENAPI_DESCRIPTOR = apis/pipeline/pipeline.yaml
 BUILD_DIR ?= build
 BUILD_PACKAGE = ${PACKAGE}/cmd/pipeline
 TEMPORARY_DIRECTORY = tmp
-VERSION ?= $(shell git describe --tags --exact-match 2>/dev/null || git symbolic-ref -q --short HEAD)
+VERSION ?= $(shell git describe --tags --exact-match 2>/dev/null || echo "`git describe --tags`-`git symbolic-ref -q --short HEAD 2>/dev/null || git describe --all --exact-match | sed 's!heads/!!'`")
```

Comment:
Adding `${GITHUB_REF#refs/heads/}` temporarily should be enough for now. (Probably needs a build arg in the dockerfile too)